risc0/build/src/docker.rs

```diff
@@ -205,6 +265,39 @@ fn compute_image_id(elf_path: &Path) -> Result<String> {
     Ok(image.compute_id().to_string())
 }
 
+/// Check if docker is running.
+fn check_docker(docker: impl AsRef<std::ffi::OsStr>, max_elapsed_time: Duration) -> Result<()> {
+    eprintln!("Checking if Docker is running...");
+    let backoff = backoff::ExponentialBackoffBuilder::new()
```

Comment:
I don't think we need to do backoff for checking if docker is running. What's the thinking behind this?

Reply:
The backoff was originally added because we first had an initial check to see if docker is running, and if it's not then we'd try to start the engine, which could take a while hence the backoff when doing the checks. 

I realize this was a bad idea and the whole docker check is already reverted.